index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
-          <p class="issue" data-number="15"></p>
-          <p>It MAY have a <dfn>Contextual Text Language</dfn> property, representing the language of the <a>Contextual Text</a> content.</p>
-        </section>
-
-        <section>
-          <h4>Event Description</h4>
-          <p>Each <a>Event</a> MAY have an <dfn>Event Description</dfn> property, as a human-readable description of the <a>Event</a>. </p>
-          <p class="issue" data-number="30"></p>
-          <p class="note">It does not have to be unique for this <a>Event</a>. It may be used to identify in a human-readable way one or more <a>Events</a> that need to be processed together, e.g. in a batch recording.</p>
-          <p class="issue" data-number="20"></p>
+          <p>A <a>Text</a> object is represented with a <code>p</code> element with the following constraints:
+            <ul>
+              <li>The text content of the <code>p</code> is the <a>Text</a> of the <a>Event</a>. It MAY also use TTML markup to represent styles.
+                <div class="note">The text content of the paragraph may contain TTML elements such as <code>span</code> or TTML attributes such as <code>tts:ruby</code> used to alter the layout or styling of each paragraph</div>
+              </li>
+              <li>The <code>p</code> element MUST have an <code>xml:lang</code> attribute corresponding to the language of the <a>Text</a> object.</li>
+          <pre class="example">
+&lt;div xml:id=&quot;event_0130df43-5443-4206-b1d9-d331e42b5a86&quot;
+     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
+     style=&quot;style_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;
+     ttm:agent=&quot;character_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;&gt;
+  &lt;p xml:lang=&quot;pt-BR&quot; &gt;Voc&#xea; vai ter.&lt;/p&gt;
+  &lt;p xml:lang=&quot;fr&quot; ttm:role=&quot;x-context&quot;&gt;Bah, il arrive.&lt;/p&gt;
+&lt;/div&gt;
+          </pre>
         </section>
 
         <section>
           <h4>On Screen</h4>
-          <p>Each <a>Event</a> MAY have a <dfn>On Screen</dfn> property, which is an annotation indicating the position of the <a>character</a> during the <a>Event</a>:
+          <p>The <dfn>On Screen</dfn> property is an annotation indicating the position in the scene of the <a>character</a> during the <a>Event</a>:
             <ul>
               <li>ON - the <a>character</a> is on screen for the entire duration</li>
               <li>OFF - the <a>character</a> is off screen for the entire duration</li>
               <li>ON_OFF - the <a>character</a> starts on screen, but goes off screen at some point</li>
               <li>OFF_ON - the <a>character</a> starts off screen, but goes on screen at some point</li>
-            </ul>as a human-readable description of the <a>Event</a>. 
+            </ul>
+          </p>
+          <p>The <a>On Screen</a> property is represented as a <code>metadata</code> element with the following constraints:
```

Comment:
The On Screen property doesn't seem to be referenced by other entities in the data model.

Reply:
Let's study and try to get rid of the metadata element for example by using the attribute direclty on the div and having desc directly in the div.

Reply:
#63